totalRP3/UI/ProfileManager/ProfileManagerList.lua

```diff
@@ -0,0 +1,85 @@
+-- Copyright The Total RP 3 Authors
+-- SPDX-License-Identifier: Apache-2.0
+
+local L = TRP3_API.loc;
+
+TRP3_ProfileManagerListMixin = {};
+
+function TRP3_ProfileManagerListMixin:OnLoad()
+	BackdropTemplateMixin.OnBackdropLoaded(self);
+
+	local scrollBoxAnchorsWithBar = {
+		AnchorUtil.CreateAnchor("TOP", self.Divider, "BOTTOM", 0, -3),
+		AnchorUtil.CreateAnchor("LEFT", self, "LEFT", 6, 0),
+		AnchorUtil.CreateAnchor("RIGHT", self.ScrollBar, "LEFT", -6, 0),
+		AnchorUtil.CreateAnchor("BOTTOM", self, "BOTTOM", 0, 6),
+	};
+
+	local scrollBoxAnchorsWithoutBar = {
+		scrollBoxAnchorsWithBar[1],
+		scrollBoxAnchorsWithBar[2],
+		AnchorUtil.CreateAnchor("RIGHT", self, "RIGHT", -6, 0),
+		scrollBoxAnchorsWithBar[4],
+	};
+
+	self.ScrollView = CreateScrollBoxListLinearView();
+	ScrollUtil.InitScrollBoxListWithScrollBar(self.ScrollBox, self.ScrollBar, self.ScrollView);
+	ScrollUtil.AddManagedScrollBarVisibilityBehavior(self.ScrollBox, self.ScrollBar, scrollBoxAnchorsWithBar, scrollBoxAnchorsWithoutBar);
+end
+
+function TRP3_ProfileManagerListMixin:OnShow()
+	self.CreateButton:SetText(L.PR_CREATE_PROFILE);
+	self.ScrollBox.EmptyText:SetText(L.PR_PROFILEMANAGER_EMPTY);
+end
+
+function TRP3_ProfileManagerListMixin:GetSearchText()
+	return self.SearchBox:GetText("");
```

Comment:
Is there a reason for the empty string here

Reply:
copy/paste